pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -54823,7 +54823,7 @@ self: {
        description = "Kafka bindings for Haskell";
        license = stdenv.lib.licenses.mit;
        hydraPlatforms = stdenv.lib.platforms.none;
-     }) { rdkafka = null;};
```

Comment:
Please don't edit this file manually! See the comment in the first line; changes you make here will be lost the next time this file is re-generated.